lib/api/tenure/v1/types.ts

```diff
@@ -65,36 +70,45 @@ export interface LegacyReference {
   name: string;
   value: string;
 }
-
+export interface FurtherAccountInformation {
+  isRentAccountRequired: boolean;
+  noRentAccountReason: string;
+  rentLetterSentDate: string;
+  rentCardGivenDate: string;
+  tenureAcceptedDate: string;
+  isSection208NoticeSent: boolean;
+}
 export interface Tenure {
   id: string;
+  paymentReference: string;
+  householdMembers: HouseholdMember[];
   tenuredAsset: TenureAsset;
+  charges: Charges;
   startOfTenureDate: string;
   endOfTenureDate: string | null;
   tenureType: TenureType;
+  tenureSource: string | null;
   isActive: boolean;
-  accountType: AccountType;
```

Comment:
Should this be removed? - We should ensure that it's not being used by any MFEs before removing it as it can cause issues

Reply:
Added all back in. I can't see it being used anywhere, but as you say quite risky